crates/rpc/src/pathfinder/methods/get_proof.rs

```diff
@@ -16,6 +17,13 @@ pub struct GetProofInput {
     pub keys: Vec<StorageAddress>,
 }
 
+#[derive(Deserialize, Debug, PartialEq, Eq)]
+pub struct GetProofInputClass {
+    pub block_id: BlockId,
+    pub class_hash: ClassHash,
+    pub keys: Vec<StorageAddress>,
```

Comment:
Here the `keys` parameter does not make sense. `pathfinder_getProof` returns the global state proof for a single contract and storage proofs for each key requested.

For the class proof, we do not have a notion of keys: we just want the proof that the class is defined in the class tree. You can just remove this parameter.